manifest.json

```diff
@@ -100,6 +98,10 @@
         "type": "string",
         "description": "admin/store.searchTermPath.description"
       },
+      "includeTwitterMetaTags": {
+        "title": "Include Twitter meta tags",
```

Comment:
Use a message ID here so that this `title` is translated across all languages in the Admin.

Reply:
I'll open the PR to the `admin-pages` app to include the updated messages.